CHANGES.md

```diff
@@ -1,6 +1,473 @@
+* added support for joinPath(), isAbsPath(), commonPath() to the `Path` module
+  (see https://chapel-lang.org/docs/1.17/modules/standard/Path.html)
+* added support for channel.advancePastByte() to read until a particular byte
+  (see https://chapel-lang.org/docs/1.17/modules/standard/IO.html#IO.channel.advancePastByte)
+* added versions of channel.mark(), commit(), revert() when locking==false
+  (see https://chapel-lang.org/docs/1.17/modules/standard/IO.html#IO.channel.mark)
+* improved support for I/O on `date`, `time`, and `datetime` types in `DateTime`
+  (see https://chapel-lang.org/docs/1.17/modules/standard/DateTime.html#DateTime.date.readWriteThis)
+* made DateTime.time.tzinfo `Shared` for automated memory management
+  (see https://chapel-lang.org/docs/1.17/modules/standard/DateTime.html#DateTime.time.init)
+* array `push*` methods now use `in` intents to improve their use with Owned
+  (see https://chapel-lang.org/docs/1.17/builtins/internal/ChapelArray.html#ChapelArray.push_back)
+* added `string.size` overload for `string.length`
+  (see https://chapel-lang.org/docs/1.17/builtins/internal/String.html#String.string.size)
+* updated several modules to use error handling rather than `try!` / `halt()`
+* made the `Buffers` module into a package module
+  (see https://chapel-lang.org/docs/1.17/modules/packages/Buffers.html)
+* made the `UtilReplicatedVar` module into a package module `ReplicatedVar`
+  (see https://chapel-lang.org/docs/1.17/modules/packages/ReplicatedVar.html)
+
+Package Modules
+---------------
+* `LinearAlgebra` matrices now use 1-based indices by default
+  (see https://chapel-lang.org/docs/1.17/modules/packages/LinearAlgebra.html#linear-algebra-interface)
+* added Kronecker product, `kron()`, to `LinearAlgebra` module
+  (see https://chapel-lang.org/docs/1.17/modules/packages/LinearAlgebra.html#LinearAlgebra.kron)
+* added Cholesky factorization routine to `LinearAlgebra` module
+  (see https://chapel-lang.org/docs/1.17/modules/packages/LinearAlgebra.html#LinearAlgebra.cholesky)
+* added eigenvalue/eigenvector solvers to `LinearAlgebra` module
+  (see https://chapel-lang.org/docs/1.17/modules/packages/LinearAlgebra.html#LinearAlgebra.eigvals)
+* added support for `eye()` to `LinearAlgebra` for CS* domains
+  (see https://chapel-lang.org/docs/1.17/modules/packages/LinearAlgebra/Sparse.html#Sparse.eye)
+* added the Blowfish cipher to the `Crypto` module
+  (see https://chapel-lang.org/docs/1.17/modules/packages/Crypto.html#Crypto.Blowfish)
+* added a new scalable barrier across all locales
+  (see https://chapel-lang.org/docs/1.17/modules/packages/AllLocalesBarriers.html)
+* added binaryInsertionSort to the `Sort` module
+  (see https://chapel-lang.org/docs/1.17/modules/packages/Sort.html#Sort.binaryInsertionSort)
+* added `--debugTomlReader` flag for `TOML.TomlReader` submodule
+  (see https://chapel-lang.org/docs/1.17/modules/packages/Sort.html#Sort.binaryInsertionSort)
```

Comment:
Sorry to catch post-merge, but I just noticed this link is incorrect. The `TOML` module is not publicly documented, so there is no associated link for this flag.

Reply:
Oops, cut and paste error on my part—thanks for catching it.